build-formula.sh

```diff
@@ -13,6 +13,13 @@ export HOMEBREW_NO_INSTALL_CLEANUP=1
 # Commenting out the auto upgrade since appveyor mac has an old version and fails buildilng bottles for SAM CLI
 # export HOMEBREW_NO_AUTO_UPDATE=1
 
+if [[ "$OSTYPE" == "linux-gnu"* ]]; then
+    test -d ~/.linuxbrew && eval "$(~/.linuxbrew/bin/brew shellenv)"
+    test -d /home/linuxbrew/.linuxbrew && eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
+    test -r ~/.bash_profile && echo "eval \"\$($(brew --prefix)/bin/brew shellenv)\"" >>~/.bash_profile
+    echo "eval \"\$($(brew --prefix)/bin/brew shellenv)\"" >>~/.profile
+fi
```

Comment:
I assume we are trying to setup brew shell configuration here. Can we put some comments on top about what this piece is trying to achieve?

Reply:
This was needed for local running in some cases. Removed as this is not needed for AppVeyor.